application/service.go

```diff
@@ -16,8 +16,10 @@ import (
 )
 
 const (
-	// TokenName is used to define form parameter name.
-	TokenName = "csrftoken"
+	// FormTokenName is used to define HTML form input field parameter name.
+	FormTokenName = "csrftoken"
+	// HeaderTokenName is used to define the header field name which can contain the token.
+	HeaderTokenName = "Csrf-Token"
```

Comment:
Expected standard name would be "X-CSRF-TOKEN" (like in this case: https://laravel.com/docs/5.8/csrf#csrf-x-csrf-token).

Reply:
Actually _X-_ prefix is deprecated (see [RFC6648 3.3](https://tools.ietf.org/html/rfc6648#section-3)